File: app/src/main/java/tk/leopro/petzyandroid/interfaces/RequestPlaceInterface.java
package tk.leopro.petzyandroid.interfaces;


import retrofit2.http.GET;
import retrofit2.http.Query;
import rx.Observable;
import tk.leopro.petzyandroid.pojo.GooglePredictionData;

//interface for retrofit

public interface RequestPlaceInterface {

    @GET("json?")
    Observable<GooglePredictionData> getJSON(@Query("query") String query, @Query("location") String location
            , @Query("radius") String radius, @Query("language") String language, @Query("key") String key);
}

File: app/src/main/java/tk/leopro/petzyandroid/interfaces/RequestPlaceInterface.kt
package tk.leopro.petzyandroid.interfaces


import retrofit2.http.GET
import retrofit2.http.Query
import rx.Observable
import tk.leopro.petzyandroid.pojo.GooglePredictionData

//interface for retrofit

interface RequestPlaceInterface {

    @GET("json?")
    fun getJSON(@Query("query") query: String, @Query("location") location: String, @Query("radius") radius: String, @Query("language") language: String, @Query("key") key: String): Observable<GooglePredictionData>
}